library/std/src/os/unix/process.rs

```diff
@@ -378,6 +378,40 @@ impl ExitStatusExt for process::ExitStatusError {
     }
 }
 
+#[unstable(feature = "unix_send_signal", issue = "141975")]
+pub trait ChildExt: Sealed {
+    /// Sends a signal to a child process.
+    ///
+    /// # Errors
+    ///
+    /// This function will return an error if the signal is invalid. The integer values associated
+    /// with signals are implemenation-specific, so the use of [`libc`] is encouraged.
```

Comment:
```suggestion
    /// with signals are implemenation-specific, so the use of `libc` is encouraged.
```

`libc` is not a public dependency so linking won't work. Also it might be worth saying something more generic. There are other crates that provide posix bindings (e.g. for safety or other reasons). They might use `libc` internally but the user doesn't necessarily need to know that.

Reply:
fixed